### ISSUE TITLE
Upgrade to the latest kotlin version

### ISSUE THREAD

**Issue by wayfarer-rus** [Dominaezzz/kgl#12]
Current version of abi is far behind the latest stable.

Could you please upgrade it?

**Comment by Dominaezzz** [comment]
Thanks for the interest in this project.
I've been meaning to upgrade it but `kotlinx-io` was taking it's sweet time to catch up to `1.3.50`. It caught up yesterday so I will be able to upgrade this soon.

**Comment by Dominaezzz** [comment]
Version `0.1.6` is built with Kotlin `1.3.50`.

**Comment by wayfarer-rus** [comment]
Great! Much appreciated. But it's not over yet =)

> Could not find com.kgl:kgl-glfw-macosx64:0.1.6.
> Could not find com.kgl:kgl-opengl-macosx64:0.1.6.

macos build is not available.

Append: 
Oh dear! 

URL https://dl.bintray.com/dominaezzz/kotlin-native/com/kgl/kgl-glfw-macosx64/0.1.6/kgl-glfw-macosx64-0.1.6.pom   is 404

but here https://bintray.com/dominaezzz/kotlin-native/kgl/0.1.6#files/com%2Fkgl%2Fkgl-glfw-macosX64%2F0.1.6  files exists

How could it be?

**Comment by Dominaezzz** [comment]
Hmm, bintray publishing seems to fail at random. Will have to further investigate this (Might be some race condition, not sure).
`0.1.6-dev-4` published just fine, so that can be used as a work-around for now.

**Comment by wayfarer-rus** [comment]
Nope =(

> Could not find com.kgl:kgl-glfw-macosx64:0.1.6-dev-4
> Could not find com.kgl:kgl-opengl-macosx64:0.1.6-dev-4

all 0.1.6-* versions produce such kind of error

0.1.5 works fine though

**Comment by Dominaezzz** [comment]
~The mac os target is called `macosX64` and not `macosx64`. Did you specify this explicitly or did you just depend on common?~
Nvm, I just checked, metadata seems to be wrong.

**Comment by Dominaezzz** [comment]
In the [module file](https://dl.bintray.com/dominaezzz/kotlin-native/com/kgl/kgl-opengl/0.1.6/kgl-opengl-0.1.6.module), it says,
```json
....,
{
  "name": "linuxX64-api",
  "attributes": {
    "org.gradle.usage": "kotlin-api",
    "org.jetbrains.kotlin.native.target": "linux_x64",
    "org.jetbrains.kotlin.platform.type": "native"
  },
  "available-at": {
    "url": "../../kgl-opengl-linuxX64/0.1.6/kgl-opengl-linuxX64-0.1.6.module",
    "group": "com.kgl",
    "module": "kgl-opengl-linuxX64",
    "version": "0.1.6"
  }
},
{
  "name": "macosX64-api",
  "attributes": {
    "org.gradle.usage": "kotlin-api",
    "org.jetbrains.kotlin.native.target": "macos_x64",
    "org.jetbrains.kotlin.platform.type": "native"
  },
  "available-at": {
    "url": "../../kgl-opengl-macosx64/0.1.6/kgl-opengl-macosx64-0.1.6.module",
    "group": "com.kgl",
    "module": "kgl-opengl-macosx64",
    "version": "0.1.6"
  }
},
....
```

Only the macos ones have been lowercased.

**Comment by wayfarer-rus** [comment]
Ok. I will continue with this project after short vacation. Currently can't check anything.

**Comment by wayfarer-rus** [comment]
Experiencing same issue even with 'macosx64' instead of 'macosX64':

> Could not find com.kgl:kgl-glfw-macosx64:0.1.6.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/kgl/kgl-glfw-macosx64/0.1.6/kgl-glfw-macosx64-0.1.6.module
       - https://repo.maven.apache.org/maven2/com/kgl/kgl-glfw-macosx64/0.1.6/kgl-glfw-macosx64-0.1.6.pom
       - https://dl.bintray.com/dominaezzz/kotlin-native/com/kgl/kgl-glfw-macosx64/0.1.6/kgl-glfw-macosx64-0.1.6.module
       - https://dl.bintray.com/dominaezzz/kotlin-native/com/kgl/kgl-glfw-macosx64/0.1.6/kgl-glfw-macosx64-0.1.6.pom
     Required by:
         project : > com.kgl:kgl-glfw:0.1.6
   > Could not find com.kgl:kgl-opengl-macosx64:0.1.6.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/kgl/kgl-opengl-macosx64/0.1.6/kgl-opengl-macosx64-0.1.6.module
       - https://repo.maven.apache.org/maven2/com/kgl/kgl-opengl-macosx64/0.1.6/kgl-opengl-macosx64-0.1.6.pom
       - https://dl.bintray.com/dominaezzz/kotlin-native/com/kgl/kgl-opengl-macosx64/0.1.6/kgl-opengl-macosx64-0.1.6.module
       - https://dl.bintray.com/dominaezzz/kotlin-native/com/kgl/kgl-opengl-macosx64/0.1.6/kgl-opengl-macosx64-0.1.6.pom
     Required by:
         project : > com.kgl:kgl-opengl:0.1.6


Could you please take a look once again?

**Comment by Dominaezzz** [comment]
I've made a work-around and published it in `0.1.8-dev-1`. See if that works.

**Comment by wayfarer-rus** [comment]
Ha-ha! It works! 

Thanks a lot!
